app/src/gql/index.js

```diff
@@ -186,7 +186,6 @@ export const ProposalFragment = `
     picture
   }
   summary
-  changes
```

Comment:
beneficiary is still here as a separate object?

Reply:
yes

Reply:
i guess i'm wondering why it's separate from description

Reply:
I see. The difference is that we have there the whole Object, not only the address. Maybe because of this it is better to leave it there for now